### ISSUE TITLE
adb-free without ORDS and APEX

### ISSUE THREAD

**Issue by alexesca** [oracle/adb-free#26]
Can you guys have an image or image's tag that has the autonomous database without apex and ords?

We already have this docker container image to install/run ords and apex on the database it connects to:

`docker pull container-registry.oracle.com/database/ords-developer:latest`

You are probably familiar with the docs here:

[ORDS Developer Docker Image](https://container-registry.oracle.com/ords/ocr/ba/database/ords-developer)

The coupling of the autonomous database and ords and apex is preventing us from embracing this image as our development image. If you guys give us a tag or image with the autonomous database setup, we wouldn't need to come to you every time there is a new apex version. 

adb-free is the closest oracle database image to the OCI autonomous database. Being able to develop our applications on this image would improve our development experience. In addition, we currently have to use a different database image and install things that are already included in the autonomous database.

**Comment by spanchum** [comment]
The goal of ADBS container image to provide the experience similar to what you get with cloud service. We are trying offer all components as much we can. You always have a choice to have your own ORDS pointing to ADBS container. Is the issue with ORDS/APEX not getting frequently updated in container image? We are working on another offering of container image with more frequent updates soon.

**Comment by alexesca** [comment]
> You always have a choice to have your own ORDS pointing to ADBS container

@spanchum I can point it to ADBS container, but the my ORDS container needs a sys connection to install or update APEX

Can I install or update APEX on ADBS without having SYS access?

**Comment by alexesca** [comment]
> Is the issue with ORDS/APEX not getting frequently updated in container image?

@spanchum Yes. We want  to be developing on the latest version of APEX and ORDS

**Comment by alexesca** [comment]
@spanchum  and @aosingh how is this going? What is the name of the new offering and when is it gonna be available?

> "We are working on another offering of container image with more frequent updates soon."

**Comment by aosingh** [comment]
@alexesca 

Currently, the Developer image offering is waiting to get a Security sign-off. We hope to have it released early next year.
